docs/operator-manual/argocd-cm.yaml

```
@@ -261,6 +261,11 @@ data:
   # - Files may also be loaded from remote locations via fully qualified URLs.
   ui.cssurl: "./custom/my-styles.css"
 
+  # Logo rendered on the dashboard pages, at top of the sidebar
+  ui.dashboardlogo: "https://mysite.com/static/logo.png"
+  # Logo rendered on the login page, at center of left section
+  ui.loginlogo: "https://mysite.com/static/logo-hero.png"
```

```suggestion
  # Make sure the source of your logo is trusted and well-controlled. Some image formats (like SVG) may
  # be security threat vectors.

  # Logo rendered on the dashboard pages, at top of the sidebar
  ui.dashboardlogo: "https://mysite.com/static/logo.png"
  # Logo rendered on the login page, at center of left section
  ui.loginlogo: "https://mysite.com/static/logo-hero.png"
```

Justification: https://security.stackexchange.com/a/135636/155615